### PR TITLE
Cassandra and postgresql interceptors

### LINKED ISSUE #159: Create Cassandra interceptor
... and do basic FI

### PR DESCRIPTION
**Pull request by MichaelA-ops**
Closes #159 and #169

### CODE REVIEW

File: src/test/java/cloud/filibuster/functional/java/cassandra/JUnitCassandraAccessTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class JUnitCassandraAccessTest extends JUnitAnnotationBaseTest {
```
Comment (cmeiklejohn) [inline src/test/java/cloud/filibuster/functional/java/cassandra/JUnitCassandraAccessTest.java:17]: Unit or integration test?

File: src/test/java/cloud/filibuster/functional/java/postgresql/JUnitDynamicProxyInterceptorTest.java [hunk 2]
```diff
@@ -62,15 +73,15 @@ public void testInterceptConnection() {
 
     @DisplayName("Verify correct number of test executions.")
     @Test
-    @Order(2)
+    @Order(3)
     public void testNumExecutions() {
-        // 1 fault free execution + 1 fault injected execution
-        assertEquals(2, numberOfTestExecutions);
+        // 2 fault free execution + 2 fault injected execution
```
Comment (cmeiklejohn) [inline src/test/java/cloud/filibuster/functional/java/postgresql/JUnitDynamicProxyInterceptorTest.java:78]: What's the reason for this change?  I don't get why there are two fault free executions?  Is this because you're testing two things?  Then, if so, these should be two different tests!

**Comment by cmeiklejohn** [comment]
Needs conflict resolution too.

File: src/main/java/cloud/filibuster/junit/configuration/examples/db/cassandra/CassandraSingleFaultOverloadedExceptionAnalysisConfigurationFile.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CassandraSingleFaultOverloadedExceptionAnalysisConfigurationFile implements FilibusterAnalysisConfigurationFile {
+    private static final FilibusterCustomAnalysisConfigurationFile filibusterCustomAnalysisConfigurationFile;
+
+    private static Map<String, String> createErrorMap() {
+        Map<String, String> myMap = new HashMap<>();
+        myMap.put("cause", "");
```
[the remark this answers is not in the document]
Comment (cmeiklejohn) [reply src/main/java/cloud/filibuster/junit/configuration/examples/db/cassandra/CassandraSingleFaultOverloadedExceptionAnalysisConfigurationFile.java:15]: Resolved.

File: src/test/java/cloud/filibuster/functional/java/postgresql/JUnitDynamicProxyInterceptorTest.java (hunk 2 above)
Comment (MichaelA-ops) [reply src/test/java/cloud/filibuster/functional/java/postgresql/JUnitDynamicProxyInterceptorTest.java]: Since CockraochDB and native PostgreSQL both use PGSimple, I just wrote one test for both of them. I've just split this test into two, one for each.

**Review by cmeiklejohn — APPROVED** [review]
